### ISSUE TITLE
BUG - sev-3 - All (kind of just iOS) - Rx: portions of multi-line Rx statuses cut off with large font (worse in list than details)

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#3943]
## What happened?

With a large font (iPhone):

In the prescription list, some of the letters in the label tag extend outside of the label tag borders (looks like "ctive" instead of "Active", for instance):
![image.png](https://images.zenhubusercontent.com/61a5397dac83844bf079e942/1d29c63f-0610-46de-8e13-b247cbe055e9)

In prescription details:
![image.png](https://images.zenhubusercontent.com/61a5397dac83844bf079e942/98a7be6c-deb8-4538-afe3-4c29ad91d861)

It looks worse in dark mode because the letters disappear because it's black font on black background (as compared to light mode, where they are fully visible but just extend outside of the pill container).

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Steps to Reproduce

Log in. Toggle on prescriptions in Profile > Settings > Developer Screen > Remote Config > Prescriptions toggle (at bottom) > Apply overrides
Log in again, as karsyn (our only Rx test user)
Health > Prescriptions (for the list) and then > get prescription details (for med details).

## Desired behavior

Text should stay within the bounds of the pill no matter the font size

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Issue Tracking](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/testing/VA%20Mobile%20App%20Test%20Plan.md#issue-tracking) for details on Severity Levels

- 4 - Low

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, feature, bug)
- [ ] Linked to an Epic (General Tech/Design Debt & Maintenance)

**Comment by DJUltraTom** [comment]
looking much better now, QA approves this change
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd3FVQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--096a3cae43e8fb75ad02980fa7cd5ec09a22e49d/IMG_0016%203.PNG)

**Comment by DJUltraTom** [comment]
looks like something happened between the on demand build and the overnight build, 
The text for "process" has changed from being capitalized to lowercase which is putting a portion of the text outside of the pill (see screenshot)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNEdoQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2a91978c4020fef06fa29753622c4ee86836180f/IMG_C2BE0FCD4974-1.jpeg)

**Comment by DJUltraTom** [comment]
after conferring with @Sparowhawk, moving this ticket back to "in-progress"

**Comment by DJUltraTom** [comment]
Cannot repro the issue. looking good!
QA approves this

**Comment by DJUltraTom** [comment]
spot checked on overnight dev build, looks good,
Closing as deployed
